core/lowering/passes/op_aliasing.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/csrc/jit/passes/subgraph_rewrite.h>
+
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace lowering {
+namespace passes {
+
+void AliasOperators(std::shared_ptr<torch::jit::Graph>& graph) {
```

Comment:
@narendasan All the alias op patterns (if there are any in future), should make it here in one place right ? If not, we should probably rename the AliasOperators to TrueDivideLowering maybe ?

Reply:
Yeah the idea is that we'd list all direct 1:1 aliases in this pass. Id like to build some infra around it so it can be defined by a map or something (#306)

Reply:
Sounds good